net-dns/idnkit/files/idnkit-2.3-incompatible-pointers.patch

```diff
@@ -0,0 +1,29 @@
+Fix incompatible pointers for modern C, hope that restrict qualifier
```

Comment:
FWIW, you can generally _add_ `const`, `restrict` and so on (as long as it's correct/appropriate, of course), but _stripping_ is UB (I think for `const`, you can strip and then put it back, but you get the point).

Reply:
It wasn't comment on the patch, it was comment on the remaining code, but noted.